src/stdafx.h

```diff
@@ -81,29 +82,62 @@
 
 #include "zip.h"
 
+namespace fs = std::tr2::sys;
```

Comment:
Let's just use Boost.FS everywhere. No particular reason to try to jiggle two subtly different implementations.